### PR TITLE
Queue monitor

### PR DESCRIPTION
**Pull request by itsbriany**
No alerts yet, just queue monitoring.

### CODE REVIEW

**Comment by imbstack** [comment]
Unfortunately this has conflicts! Otherwise looks good. r+

File: test/api_test.js [hunk 1]
```diff
@@ -21,14 +21,14 @@ suite('API', () => {
     await namespaces.removeTable();
   });
 
-  test('ping', () => {
+  test.skip('ping', () => {
```
Comment (itsbriany) [inline test/api_test.js]: It looks like `api` does not have a ping method anymore. For now, I just skipped this test, but if this functionality is not necessary anymore, I think the `ping` test is no longer necessary.

Comment (itsbriany) [inline test/api_test.js]: I rebased master onto this branch, but it looks like I cannot find any object which has a `ping` method on it, hence this test is still failing for me :(

Which file holds the JS class defining the `ping` method?

**Comment by Hexcles** [comment]
It is implemented in the new version of the base class,
taskcluster-lib-api, so now each individual API class no longer needs to
implement ping itself. The change that updated the dependency version and
removed ping was recently merged.

Maybe try running npm install to update the base class.

File: test/api_test.js (hunk 1 above)
Comment (itsbriany) [inline test/api_test.js]: @Hexcles Thanks, that explains why my tests were failing :) Only thing now is when I run the tests, I get a verbose output "Authorized test-client for POST access..." when I run the API tests. I was wondering if there is a way to disable this for testing only?

**Comment by itsbriany** [comment]
@Hexcles Thanks! It looks like when I run the API tests, I now get a verbose output: "Authorized test-client for POST access to...". I can see this console message being useful for when we run the application, but it would be nice to suppress those messages while testing.

**Comment by djmitche** [comment]
There's not currently a way to disable that at the moment, but it's not a bad idea to do so.

**Pull request merged**